Kernel/Syscalls/jail.cpp

```diff
@@ -24,14 +36,25 @@ ErrorOr<FlatPtr> Process::sys$jail_create(Userspace<Syscall::SC_jail_create_para
     if (jail_name->length() > jail_name_max_size)
         return ENAMETOOLONG;
 
+    // NOTE: Either the default isolation flag is set or individual flags are set
+    if (params.flags & static_cast<unsigned>(JailIsolationFlags::Default)
+        && ((params.flags & ~static_cast<unsigned>(JailIsolationFlags::Default)) != 0))
```

Comment:
This means `::None` is an invalid flag - either we should support that flag, or we should not have it in the first place?

Reply:
Ah, I meant that if you don't set anything then that's actually OK, but if you set the default flag with other flags this is invalid. I will clarify this.